### PR TITLE
Wikibase configurations for gpcommonswiki

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: Wikibase.php [hunk 1]
```diff
@@ -110,7 +110,27 @@
 if ( $wgDBname === 'gpcommonswiki' ) {
 	$wgWBClientSettings['repoSiteName'] = 'Gratisdata';
 	$wgWBClientSettings['pageSchemaNamespaces'] = [ 0 ];
-
+		// Some well-known properties' IDs which are used to format references
```
Comment (Universal-Omega) [inline Wikibase.php:113]: Tabbing

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by ugochimobi** [comment]
@Universal-Omega Fixed

**Comment by Universal-Omega** [comment]
Conflicts

**Comment by ugochimobi** [comment]
Done

**Review by Universal-Omega — APPROVED** [review]

**Pull request merged**